Makefile

```diff
@@ -61,3 +61,20 @@ binaries: ${PREFIX}/bin/registry ${PREFIX}/bin/registry-api-descriptor-template
 clean:
 	@echo "+ $@"
 	@rm -rf "${PREFIX}/bin/registry" "${PREFIX}/bin/registry-api-descriptor-template"
+
+# import the existing docs build cmds from docker/docker
```

Comment:
Could this section be isolated in an include? Something like `doc/doc.mk`. I see the docs build being external to this makefile at some point in the future.